src/dirtyfields/dirtyfields.py

```diff
@@ -133,14 +133,16 @@ def reset_state(sender, instance, **kwargs):
     # getting a `KeyError` when checking if a field is dirty or not
     update_fields = kwargs.pop('update_fields', {})
     new_state = instance._as_dict(check_relationship=True)
+    FIELDS_TO_CHECK = getattr(instance, "FIELDS_TO_CHECK", None)
     if update_fields:
         for field_name in update_fields:
-            field = sender._meta.get_field(field_name)
+            if not FIELDS_TO_CHECK or field_name in FIELDS_TO_CHECK:
```

Comment:
Hello @bparker98 !
Does this work well with FKs?
For example, with this model definition:
```
class TestModelWithFk(Model):
    my_field = models.ForeignKey(...)
    FIELDS_TO_CHECK = ['my_field_id']
```
I think you may have to check `field.get_attname()` instead of `field_name`, like this:
https://github.com/romgar/django-dirtyfields/blob/ef4e401feb011abee120c11f4097818fa2e04fbd/src/dirtyfields/dirtyfields.py#L43